src/main/java/graphql/schema/diffing/ana/EditOperationAnalyzer.java

```diff
@@ -932,7 +933,7 @@ private void typeEdgeInsertedForInputField(EditOperation
             return;
         }
         String newType = getTypeFromEdgeLabel(editOperation.getTargetEdge());
-        EditOperation deletedTypeEdgeOperation = findDeletedEdge(inputField, editOperations, mapping);
+        EditOperation deletedTypeEdgeOperation = findDeletedEdge(inputField, editOperations, mapping, edge -> edge.getLabel().contains("type="));
```

Comment:
Is "type=" strings a good idea here?

Reply:
Well I reckon a better solution is to store a relationship enum that tracks how the two vertices were related to each other.

Andi suggested this is fine for now. Or did I misunderstand in our meeting? @andimarek

Reply:
Yes the edge having `type=` as indicator that this is an "type edge" is ok for now.

But: can we make this is bit more dry and contain this logic a bit?

We have already `getDefaultValueFromEdgeLabel` helper method. I would recommend to add another helper method like "isTypeEdge" and use that.

Reply:
Good point